11-model-comparison/11_model_comparison.ipynb

```diff
@@ -0,0 +1,462 @@
+{
```

Comment:
<p>the question in my mind regarding bayesian models as I finish this is overwhelmingly, "okay, but can you show how it actually works?" Among others--</p><p> </p><p>This whole section could use more beef/breathing room, actually. If I'm a high schooler and I don't know any statistics, this will all be pretty meaningless to me. "What the hell is a frequentist model and why should I care," thinks the high schooler. How exactly does a frequentist model fail on skewed, multimodal, or otherwise ugly data? why is a Bayesian model better? can you show how mean squared error underestimate skewness, etc?  What if I don't even know what skew is? (You can easily show that with a picture, though.) Why would I care that messing around with residuals and estimators is problematic if I don't even know what those are? What's an objective function, and why should I care?</p><p> </p><p>Since your conclusion here is an argument in favor of Bayesian models, you should put more emphasis on Bayes theorem in the first place. And/or stick to scoring metrics only here--and/or explicitly compare to other types of models. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TuringLang/TuringTutorials/pull/275/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>